### PR TITLE
Updated Schema

### PR DESCRIPTION
**Pull request by RemusCM**
Updated schema with foreign keys, use this new script for your databases. For it to work, you will probably need to manually insert some values first to get the full functionality as before.

### CODE REVIEW

**Comment by CrispiChicken** [comment]
Sorry if it was asked before but why do the names of the branches start with lower case now? Do I need to change that in my Pr with population scripts?

**Comment by RemusCM** [comment]
> Sorry if it was asked before but why do the names of the branches start with lower case now? Do I need to change that in my Pr with population scripts?


Yeah sorry, it's the way I put it on the school server. Either way would've worked, but that's how I ended up doing it lol. Could you switch to lower case for table names? If not, I can do it.^^

**Comment by CrispiChicken** [comment]
> > Sorry if it was asked before but why do the names of the branches start with lower case now? Do I need to change that in my Pr with population scripts?
> 
> Yeah sorry, it's the way I put it on the school server. Either way would've worked, but that's how I ended up doing it lol. Could you switch to lower case for table names? If not, I can do it.^^

Lol sure I'll change it later today

File: schema.sql [hunk 1]
```diff
@@ -21,84 +16,89 @@ CREATE TABLE `Admin` (
   `password` varchar(255) NOT NULL,
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```
Comment (samantha-kerr) [inline schema.sql]: Missed a table

Comment (RemusCM) [reply schema.sql]: admin is already there lol

Comment (RemusCM) [reply schema.sql]: Oh weird, that Admin table shouldn't even be there woops.

File: schema.sql [hunk 2]
```diff
@@ -21,84 +16,89 @@ CREATE TABLE `Admin` (
...
--- Structure de la table `Bills`
---
 
-CREATE TABLE `Bills` (
-  `bill_id` int(11) NOT NULL,
+DROP TABLE IF EXISTS `admin`;
+CREATE TABLE IF NOT EXISTS `admin` (
+  `admin_id` int(11) NOT NULL,
+  `password` varchar(255) NOT NULL,
+  PRIMARY KEY (`admin_id`)
```
[the remark this answers is not in the document]
Comment (RemusCM) [reply schema.sql]: That I don't know. We only have a single admin account, so it doesn't really inherit employee_id.

[the remark this answers is not in the document]
Comment (jaslatendresse) [reply schema.sql]: in the population script there are 3 records

[the remark this answers is not in the document]
Comment (jaslatendresse) [reply schema.sql]: even if we only had one account, the relationship is the same

**Pull request merged**